### DISCUSSION TITLE
A new place to share your beautiful thunderstorms!

### DISCUSSION THREAD

**Discussion by Meso22** [niels747/2D-Weather-Sandbox#88] [Show and tell]
Share your thunderstorms here!

**Comment by Meso22** [comment]
![image](https://github.com/niels747/2D-Weather-Sandbox/assets/105577122/6ff1c5da-0abe-44f1-bfd3-50a1cb9fe192)
Decent coastal thunderstorm

**Comment by Meso22** [comment]
![image](https://github.com/niels747/2D-Weather-Sandbox/assets/105577122/85e6038e-8683-4d9f-80b8-fd6b5a2cfe21)

**Comment by Meso22** [comment]
![image](https://github.com/niels747/2D-Weather-Sandbox/assets/105577122/a586a040-8c73-493b-96da-d2867826c1b5)

**Comment by qfmp1012** [comment]
Weak storm with very nice Mammatus clouds approaching shoreline
![Screenshot (11)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/a504046c-6296-4958-80f7-5652d1663d45)
and here is multiple attempts of forming an updraft ruined by wind shear
![Screenshot (13)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/cee5d6ca-ada2-46ff-8e27-56800bed173e)
![Screenshot (12)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/3b268658-ef8d-426d-ad1f-0fe5fcf2f418)

**Reply by Meso22** [reply]
Nice

**Comment by MET905** [comment]
<img width="867" alt="2" src="https://github.com/niels747/2D-Weather-Sandbox/assets/113155608/3057e954-56ef-45dc-8657-c68a27bb9588">
Mesoscale convective complex that formed at 12 am in Chicago. these storms quickly pushed their way over lake Erie and caused extreme rainfall amounts and hail. by 5 am the storms were out of the area. 

best way to form an MCC:

LOTS of moisture and heat, (not too much heat as you will cap the atmosphere and have no storms just weak showers.)
decent wind profiles and a trigger in this case storms had days of built up moisture and heat and I had a cold front spark everything off around 9 pm and as the (sim hours) went by storms organized into a giant MCC.

**Reply by qfmp1012** [reply]
how much is too much heat? like 35 degree water temp?

**Reply by MET905** [reply]
I tend to stay around 27 degrees.

**Reply by qfmp1012** [reply]
btw what does "decent wind profile" mean? is that lots of windshear or something. and what's the best brush intensity to make a good enough cold front.

**Comment by MET905** [comment]
![6](https://github.com/niels747/2D-Weather-Sandbox/assets/113155608/6faec53e-49f5-4f40-866b-42ed91ace190)
Today was supposed to be a major severe thunderstorm outbreak but the CAP was too strong and we are already at 5 pm meaning daytime heating will be lost soon, Cold front should be over the area by 8 pm look like a few showers and thats it. better luck next time :( 

WHAT A BUST!

**Comment by qfmp1012** [comment]
Severe thunderstorms created by the air being much much colder than the ocean, how ever the rising warm air melts snow before reaching the ground
![Screenshot (15)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/e3982e03-a25b-43fd-ab6b-127da25489d0)

**Comment by qfmp1012** [comment]
oh hell nah
![Screenshot (16)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/5a41024f-602d-4127-9d56-ba79e2c95410)

**Reply by MET905** [reply]
just rain for me. the cap is way too strong to support deep convection. Nice storm!

**Comment by qfmp1012** [comment]
W tornado. well not really a tornado but pretty close
![Screenshot (17)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/19c5b937-5df0-433f-a81d-547379ce50c6)
![Screenshot (18)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/2093c7cf-d5e2-4deb-b6a6-cb69ca9336b3)

**Comment by Meso22** [comment]
![image](https://github.com/niels747/2D-Weather-Sandbox/assets/105577122/a54629e4-efc1-4753-bd03-4fa3490162a0)
This is honestly beautiful

**Comment by Meso22** [comment]
![image](https://github.com/niels747/2D-Weather-Sandbox/assets/105577122/b19a8404-9fe4-4513-b7b2-e6369d33f84d)
wow, quite elevated, 3400m base

**Comment by qfmp1012** [comment]
closest i ever got to a fully-fledged tornado. just if it was more windy.
![Screenshot (19)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/0d41f7f0-1644-456a-880c-60c501cd2fe9)
![Screenshot (20)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/93008c40-cc93-4f59-8a02-596487d53fcd)

**Reply by Meso22** [reply]
That top picture reminds me of one of those low based supercells from oklahoma

**Comment by MET905** [comment]
<img width="762" alt="supercell" src="https://github.com/niels747/2D-Weather-Sandbox/assets/113155608/fb6ddfac-ea8a-440a-8d5a-3cebeedf4d96">
 
Juicy supercell!

**Reply by Meso22** [reply]
that is a wonderfully insane storm!

**Reply by qfmp1012** [reply]
Damn this looks like 20 meter deep snow at least.

**Comment by qfmp1012** [comment]
tf
![Screenshot (23)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/6a71d528-16ec-4642-af07-e118213e6b48)

**Reply by qfmp1012** [reply]
this is what happens when you tamper with the settings too much. but i got a pretty good sunset storm
![Screenshot (24)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/c5c9a743-7526-4df1-9208-eeb41a9938f8)
![Screenshot (25)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/77079bcf-29bb-4bd3-87cb-a38e017905ba)
24 degrees Celsius at an altitude of 14000 meters, seems like the stratosphere was replaced by the thermosphere.

**Comment by HurricaneHugo** [comment]
2000 mph+ winds
![Screenshot 2024-08-17 3 43 31 PM](https://github.com/user-attachments/assets/05ae3533-196d-48f0-b276-ac4d534bc543)
